global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "2.2.109"
+    "version": "3.1.100"
```

Comment:
We'll upgrade incrementally to 3.1.101 when the runtime is available in appservice.